bin/github-pages

```diff
@@ -1,54 +1,54 @@
 #!/usr/bin/env ruby
 
 require 'mercenary'
-require_relative "../lib/github-pages"
+require_relative '../lib/github-pages'
```

Comment:
I thought it was double quotes everywhere. Let's get a rubocop file going for [GitHub's Ruby Styleguide](https://github.com/styleguide/ruby)